_build/data/transport.core.system_settings.php

```diff
@@ -1969,4 +1969,22 @@
     'area' => 'site',
     'editedon' => null,
 ), '', true, true);
+$settings['uberbar'] =  $xpdo->newObject('modSystemSetting');
+$settings['uberbar_mode']-fromArrqy(array (
```

Comment:
`$settings['uberbar_mode']-fromArrqy(` should be `$settings['uberbar_mode']->fromArray(`